std_msgs/Header header
float32 dist
int32 px
int32 py